### ISSUE TITLE
Bad formatting for #reader line in teachpacks

### ISSUE THREAD

**Issue by cassanof** [sorawee/fmt#39]
Hi, the `#reader` line for DrRacket's teachpacks gets cut of by the formatter. demo:
```
;; The first three lines of this file were inserted by DrRacket. They record metadata
;; about the language level of this file in a form that our tools can easily process.
#reader(lib "htdp-intermediate-lambda-reader.ss" "lang")((modname my-file) (read-case-sensitive #t) (teachpacks ()) (htdp-settings #(#t constructor repeating-decimal #f #t none #f () #f)))
```
becomes
```
;; The first three lines of this file were inserted by DrRacket. They record metadata
;; about the language level of this file in a form that our tools can easily process.
#reader
(lib "htdp-intermediate-lambda-reader.ss" "lang")
((modname my-file) (read-case-sensitive #t)
                         (teachpacks ())
                         (htdp-settings #(#t constructor repeating-decimal #f #t none #f () #f)))
```

DrRacket needs that in all one line, or it refuses it to read it.
I could push a fix if people want this into the main branch.

**Comment by sorawee** [comment]
Please do submit a PR!

**Comment by arjunguha** [comment]
Fix here: https://github.com/cassanof/fmt/commit/8bb0794318ae7cf2867354d4b64879d282c1e16e

Thanks!

**Comment by sorawee** [comment]
Whoops, forgetting to close the issue.

Fixed by #40.
